bin/prepare-yoloe.py

```
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+from pathlib import Path
+
+import typer
+from typing_extensions import Annotated
```

Ruff (0.11.9)

Import from `typing` instead: `Annotated`

Import from `typing`

(UP035)